core/packages/test/config/launch-config.toml

```diff
@@ -0,0 +1,95 @@
+[settings]
+node_spawn_timeout = 120
+provider = "native"
+timeout = 600
+
+## Rococo
+[relaychain]
+chain = "rococo-local"
+default_command = "/tmp/snowbridge/bin/polkadot"
+default_args = ["-lparachain=debug","--beefy","--enable-offchain-indexing=true","--pruning","archive"]
+
+[relaychain.genesis.runtime.runtime_genesis_config.configuration.config]
+max_validators_per_core = 2
+needed_approvals = 1
+validation_upgrade_cooldown = 2
+validation_upgrade_delay = 2
+
+[[relaychain.nodes]]
+name = "rococo-alice"
+rpc_port = 30444
+validator = true
+ws_port = 9944
+
+[[relaychain.nodes]]
+name = "rococo-bob"
+validator = true
+
+[[relaychain.nodes]]
+name = "rococo-charlie"
+validator = true
+
+[[relaychain.nodes]]
+name = "rococo-dave"
+validator = true
+
+# Snowbridge
+[[parachains]]
+add_to_genesis = true
+chain_spec_path = "/tmp/snowbridge/spec.json"
+cumulus_based = true
+id = 1000
+
+[[parachains.collators]]
+command = "/tmp/snowbridge/bin/snowbridge"
+name = "snowbrige-alice"                              
+rpc_port = 31200
```

Comment:
The rpc port for alice is actually 8081, according to the old config. And the p2p port is 31200.